customer-terminate-service/app/src/test/java/se/inera/intyg/cts/application/service/TerminationServiceImplTest.java

```diff
@@ -130,6 +146,45 @@ void shallReturnAllExistingTerminations() {
       );
     }
 
-    assertEquals(numberOfTerminations, terminationService.findAll().size());
+    assertEquals(numberOfTerminations, terminationServiceImpl.findAll().size());
+  }
+
+  @Test
```

Comment:
Suggestion: A @Nested block around crypto key to for clarity

Reply:
Fixed!